### PR TITLE
Homework1: Andrey Sokolovsky

### PR DESCRIPTION
**Pull request by 4sokky**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework1/simple_prog10.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# 11 lines: Triple-quoted strings, while loop
+
+REFRAIN = '''
+%d bottles of beer on the wall,
+%d bottles of beer,
+take one down, pass it around,
+%d bottles of beer on the wall!
+'''
+bottles_of_beer = 9
```
Comment (belovmd) [inline src/homework1/simple_prog10.py:11]: it has sense to split constant and code with 1 empty line

File: src/homework1/simple_prog12.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# 13 lines: Unit testing with unittest
+
+import unittest
+
```
Comment (belovmd) [inline src/homework1/simple_prog12.py:4]: here you have imports and code splitted with 2 blank lines.

File: src/homework1/simple_prog14.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# 15 lines: itertools
+
+from itertools import groupby
```
Comment (belovmd) [inline src/homework1/simple_prog14.py:3]: here you have imports and code splitted with 0 blank lines. it should be consistent

File: src/homework1/simple_prog14.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+first paragraph.
+
+This is the second.
+'''.splitlines()
+# Use itertools.group by and bool to return groups of
+# consecutive lines that either have content or don't.
+for has_chars, frags in groupby(lines, bool):
+    if has_chars:
+        print(' '.join(frags))
+# PRINTS:
```
Comment (belovmd) [inline src/homework1/simple_prog14.py]: redundant comment

File: src/homework1/simple_prog16.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# 18 lines: 8-Queens Problem (recursion)
+
+BOARD_SIZE = 8
+
```
Comment (belovmd) [inline src/homework1/simple_prog16.py:4]: here is constant is separated with 2 blank lines (earlier you had 0 line split in task 10 and task14

File: src/homework1/simple_prog9.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# 10 lines: Time, conditionals, from..import, for..else
+
+from time import localtime
+
```
Comment (belovmd) [inline src/homework1/simple_prog9.py:4]: 1 line between import and code

**Review by belovmd — CHANGES_REQUESTED** [review]

File: src/homework1/simple_prog10.py (hunk 1 above)
Comment (4sokky) [reply src/homework1/simple_prog10.py:11]: if i understand correctly, i should split constant same like imports, so with 2 blank lines after them (for the consistency). right?

**Review by belovmd — APPROVED** [review]
1

**Pull request closed without merging**